modules/ROOT/pages/op-installing-tkn-on-windows.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-windows"]
+
+= Installing {op-title} on Windows
+
+== Binary installation
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-windows-amd64.exe[`tkn.exe`] file.
+. Add the location of your `tkn.exe` to the `PATH` variable.
+
+[discrete]
+== Setting the `PATH` variable for Windows 7/8
+
+The following example demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses C:\tkn-bin as the location.
+
+. Create a folder at `C:\tkn-bin`.
```

Comment:
in the `C:\tkn-bin` directory.

Reply:
removed this line since the opening sentence already talks about the example.